.mage/git.go

```diff
@@ -42,7 +43,11 @@ func (Git) installHook(name string) (err error) {
 			err = closeErr
 		}
 	}()
-	if _, err = fmt.Fprintf(f, "ARGS=\"$@\" make git.%s\n", name); err != nil {
+	mage, mageErr := filepath.Abs(os.Args[0])
+	if mageErr != nil {
+		mage = "./mage"
+	}
+	if _, err = fmt.Fprintf(f, "HOOK=\"%s\" ARGS=\"$@\" \"%s\" git:runHook\n", name, mage); err != nil {
```

Comment:
I'd assume that `./mage` should already be there, however maybe a better solution is to use `go run github.com/magefile/mage` instead of `./mage` or anything else here?

Reply:
Going with `go run github.com/magefile/mage`, as it seems like a cleaner solution